python/tests/res/enkf/test_ecl_config.py

```diff
@@ -121,4 +120,24 @@ def test_refcase( self ):
         refcaseName = ec.getRefcaseName() + ".DATA"
         self.assertEqual( dfile , refcaseName )
 
+    def test_ecl_config_constructor(self):
+        config_dict = {
```

Comment:
Please check INIT_SECTION_KEY since it parsed in ecl_config

Reply:
After discussing with Markus we decided on marking this keyword as deprecated and not include in the tests.

Reply:
ok